tests/benchmarks.py

```diff
@@ -175,7 +175,7 @@ def bench_download_50_1gb(adl, lpath, rpath, config):
             'bench_upload_50_1gb': {
                 'nthreads': 64,
                 'buffersize': 32 * 2**20,
```

Comment:
Is this still the ideal buffer size for upload? In my local one I believe I reduced it to match the block size (and I think ultimately for upload we will do the same as download and have the block stream directly, right)?